src/pedal.cpp

```diff
@@ -14,6 +14,10 @@
 //----------------------------------------------------------------------------
 
 #include "verticalaligner.h"
+#include "functorparams.h"
+#include "MidiFile.h"
```

Comment:
Includes from external libraries are listed separately (see https://github.com/rism-ch/verovio/blob/master/doc/guidelines.md#includes-and-forward-declarations)